standard/types.md

```diff
@@ -597,12 +598,25 @@ Since type parameters are not inherited, constraints are never inherited either.
 
 A type parameter is an identifier designating a value type or reference type that the parameter is bound to at run-time.
 
+type_parameter, nullable_type_parameter, non_nullable_non_value_type_parameter
+
```

Comment:
Is this a writer's note left in by accident?

```suggestion
```

Reply:
Yes.